src/Symfony/Bridge/Twig/Command/LintCommand.php

```diff
@@ -79,6 +84,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $io = new SymfonyStyle($input, $output);
         $filenames = $input->getArgument('filename');
         $showDeprecations = $input->getOption('show-deprecations');
+        $this->format = $input->getOption('format');
+
+        if ('github' === $this->format && !class_exists(GithubActionReporter::class)) {
+            throw new \InvalidArgumentException('The "github" format is only available since "symfony/console" >= 5.3.');
```

Comment:
why not bumping the min version of `symfony/console` instead ?

Reply:
See https://github.com/symfony/symfony/pull/38982#discussion_r523776928

Reply:
well, that's not consistent with the way we manage other dependencies between our own components in Symfony.

Reply:
So should I upgrade conflict dependencies?

Reply:
Same for the xliff linter command Pr please